docs/getting-started-guides/dcos.md

```diff
@@ -0,0 +1,175 @@
+
+If you want to build your own release, the package source can be found in the [Kubernetes-Mesos](https://github.com/mesosphere/kubernetes-mesos) github repository.
+
+This tutorial will walk you through setting up Kubernetes on a DCOS cluster and starting your first pod with an nginx webserver.
+
+**NOTE:** There are [known issues with the current implementation](../../contrib/mesos/docs/issues.md).
+If you have problems completing the steps below, please [file an issue against the kubernetes-mesos project](https://github.com/mesosphere/kubernetes-mesos/issues).
+
+Further information about Kubernetes on Mesos is available in the [contrib directory](../../contrib/mesos/README.md).
+
+
+## Prerequisites
+
+- A running [DCOS cluster](https://mesosphere.com/product/)
+  - [DCOS Community Edition](https://docs.mesosphere.com/install/) is currently available on [AWS](https://mesosphere.com/amazon/), [GCE](https://mesosphere.com/google/) (early access) & [Azure](https://mesosphere.com/azure/) (coming soon).
+  - [DCOS Enterprise Edition](https://mesosphere.com/product/) can be deployed on virtual or bare metal machines. Contact sales@mesosphere.com for more info and to set up an engagement.
+- [DCOS CLI](https://docs.mesosphere.com/install/cli/) installed locally
+
+
+## Install Kubernetes
+
+1. Configure and validate the Mesosphere Multiverse as a package source repository
+
+  ```
+  $ dcos config prepend package.sources https://github.com/mesosphere/multiverse/archive/version-1.x.zip
+  $ dcos package update --validate
+  ```
+
+2. Install Etcd
+
+  By default, the Kubernetes DCOS package comes with its own single-node Etcd, but in order to survive Kubernetes component process and/or container loss, an HA Etcd cluster is required.
+  Since [Etcd](https://github.com/mesosphere/etcd-mesos) can be installed on and managed by DCOS, it is the logical choice.
+
+  ```
+  $ dcos package install etcd
+  ```
+
+3. Verify that Etcd is installed and healthy
+
+  ```
+  $ dcos marathon app list
+  ```
+
+  The Etcd cluster takes a short while to deploy. Wait until it is healthy before proceeding to the next step.
+
+4. Create Kubernetes installation configuration
+
+  The Kubernetes installer must be configured to use the HA Etcd.
+
+  ```
+  $ cat >/tmp/options.json <<EOF
+  {
+    "kubernetes": {
+      "etcd-mesos-framework-name": "etcd"
+    }
+  }
+  EOF
+  ```
+
+5. Install Kubernetes
+
+  ```
+  $ dcos package install --options=/tmp/options.json kubernetes
+  ```
+
+6. Verify that Kubernetes is installed and healthy
+
+  ```
+  $ dcos marathon app list
+  ```
+
+  The Kubernetes cluster takes a short while to deploy. Wait until it is healthy before proceeding to the next step.
```

Comment:
Suggestion: "The Kubernetes cluster takes a short while to deploy. Use the DCOS UI to verify that it is healthy before going on to the next step."